[2024-03-19T00:13:24.656+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=recalls_etl_v1/run_id=manual__2024-03-19T00:13:20.215553+00:00/task_id=clean_raw_data permission to 509
[2024-03-19T00:13:24.695+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=recalls_etl_v1/run_id=manual__2024-03-19T00:13:20.215553+00:00/task_id=clean_raw_data permission to 509
[2024-03-19T00:13:24.733+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: recalls_etl_v1.clean_raw_data manual__2024-03-19T00:13:20.215553+00:00 [queued]>
[2024-03-19T00:13:24.745+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: recalls_etl_v1.clean_raw_data manual__2024-03-19T00:13:20.215553+00:00 [queued]>
[2024-03-19T00:13:24.746+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-03-19T00:13:24.769+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): clean_raw_data> on 2024-03-19 00:13:20.215553+00:00
[2024-03-19T00:13:24.775+0000] {standard_task_runner.py:57} INFO - Started process 3216 to run task
[2024-03-19T00:13:24.780+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'recalls_etl_v1', 'clean_raw_data', 'manual__2024-03-19T00:13:20.215553+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/recalls_etl_v1.py', '--cfg-path', '/tmp/tmpji2slo9p']
[2024-03-19T00:13:24.782+0000] {standard_task_runner.py:85} INFO - Job 58: Subtask clean_raw_data
[2024-03-19T00:13:24.839+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=recalls_etl_v1/run_id=manual__2024-03-19T00:13:20.215553+00:00/task_id=clean_raw_data permission to 509
[2024-03-19T00:13:24.848+0000] {task_command.py:416} INFO - Running <TaskInstance: recalls_etl_v1.clean_raw_data manual__2024-03-19T00:13:20.215553+00:00 [running]> on host 3694d3cee1cf
[2024-03-19T00:13:24.946+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='recalls_etl_v1' AIRFLOW_CTX_TASK_ID='clean_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2024-03-19T00:13:20.215553+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-19T00:13:20.215553+00:00'
[2024-03-19T00:13:24.969+0000] {logging_mixin.py:154} INFO - Success: cleaned 5 recall records.
[2024-03-19T00:13:24.969+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-03-19T00:13:24.980+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=recalls_etl_v1, task_id=clean_raw_data, execution_date=20240319T001320, start_date=20240319T001324, end_date=20240319T001324
[2024-03-19T00:13:25.032+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-03-19T00:13:25.056+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
